doc/kconfig/conf.py

```diff
@@ -0,0 +1,62 @@
+# Kconfig documentation build configuration file
+
+import os
+
+NRF_BASE = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
+
+# -- General configuration ------------------------------------------------
+
+# Sphinx 2.0 changes the default from 'index' to 'contents'
+master_doc = 'index'
+
+# General information about the project.
+project = 'Kconfig Reference'
+copyright = '2019, Nordic Semiconductor'
+author = 'Nordic Semiconductor'
+
+version = '1.0.0'  # Dummy version
```

Comment:
How do we do the versioning here? Can we just keep it unversioned, or would that be confusing?

Reply:
Not sure if it can be left out, so I just put in some version.

Does it show up in some place now? [Looks like](http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version) it's only used to replace `|version|`.

Reply:
Hmm... maybe it's used in other places too.

Could you try removing it and see what happens? Don't have a handy doc build environment available atm.

Reply:
```suggestion
version = '&nbsp;'  # Dummy version
```

Replace with non-breakable space to get rid of the version number, but keep the spacing the same as for the other doc sets.